Sources/FoundationEssentials/WinSDK+Extensions.swift

```diff
@@ -145,6 +145,18 @@ package var PAGE_READONLY: DWORD {
     DWORD(WinSDK.PAGE_READONLY)
 }
 
+package var PATHCCH_ALLOW_LONG_PATHS: ULONG {
+    ULONG(WinSDK.PATHCCH_ALLOW_LONG_PATHS.rawValue)
+}
+
+package var RRF_RT_REG_SZ: DWORD {
```

Comment:
It looks like this declaration and the one above aren't used in this change. Are they necessary?

Reply:
They are used in subsequent PRs that I have in my tree. The PRs take a while to get merged and the overloads here are just difficult to keep separated.